requirements.yml

```diff
@@ -3,9 +3,7 @@
 - src: openmicroscopy.postgresql
   version: 2.0.0
 
-- name: openmicroscopy.omero-common
-  src:  https://github.com/openmicroscopy/ansible-role-omero-common.git
-  # TODO: workaround for https://github.com/ansible/galaxy-issues/issues/259
+- src: openmicroscopy.omero-common
```

Comment:
After recent issues, should this be:

|          | cron'd                    | un-cron'd     |
|----------|---------------------------|---------------|
| pinned   | not useful?               | requires a PR |
| unpinned | picks up breaking changes | current state |

? cc: @sbesson

Reply:
Given the ongoing upstream role changes in https://github.com/openmicroscopy/ansible-role-omero-server/pull/17 and since `openmicroscopy.postgresql` is already pinned, I would minimally pin `openmicroscopy.omero-server` as well to the last stable version.

Reply:
NB: openmicroscopy/ansible-role-omero-server#17 is now merged. Propose using pinned/un-cron'd here.

Reply:
https://github.com/openmicroscopy/ansible-role-omero-server/pull/17 is untagged. You could either tag it as it is, or include https://github.com/openmicroscopy/ansible-role-omero-server/pull/18

Reply:
Done, as 2.0.0-m4.